### ISSUE TITLE
[A11y] - Dark Theme: Graphical object lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by VaniChinnam** [dequelabs/cauldron#1392]
### Current Behaviour 
`Dark Theme: Graphical object lacks 3 to 1 contrast ratio `<br/><hr/>
### Expected Behaviour 
`RULE :  Parts of graphics (required to understand the content) MUST have a contrast ratio of 3 to 1 against adjacent color(s). Exceptions exist. HOW TO FIX:  Fix this issue by adjusting the parts of graphics (that are required to understand the content) and/or adjacent color to increase the contrast to at least 3 to 1. `<br/><hr/>
### Impact 
 <p>

- [ ] Blocker
- [x] Serious
- [ ] Critical
- [ ] Moderate
- [ ] Minor

### Issue Details

<details><summary>View Issue 1658227 </summary> 
 <p>

_**Auditor ID**_ : [1658227](https://axeauditor.dequecloud.com/issue-global-search?searchKeyword=1658227&searchOn=id) 

_**description**_ :  Example bars 
Element location: Below the "Spaces" heading

Graphical bars color code: #0077CC 
Adjacent color codes: #283640 
Contrast ratio: 2.7:1 

_**Page Name**_ : Spacing 

_**source**_ : `<td class="TableCell"><div class="Spacing__example" style="padding: 16px;"></div></td>` 

_**Screenshots**_ : [Graphical object lacks 3 to 1 contrast ratio](https://axeauditor.dequecloud.com/api/v1/file/630c7d7e-bf44-11ee-bde2-fff1dcb891fe) 

</p>
</details>

**Comment by scurker** [comment]
For this one, we can just change the background color to be `var(--accent-light)` for dark mode only.
